### ISSUE TITLE
Run yamllint as a part of the PR tester

### ISSUE THREAD

**Issue by mike-nguyen** [projectatomic/atomic-host-tests#300]
In #297 and #299 yes/no values should be quoted or true/false should be used instead.  These issues were identified by yamllint.  

Lets run yamllint as part of the PR testing so ansible playbooks/roles are compliant.

**Comment by jlebon** [comment]
That makes sense to me. I'd even say let's mark it as [required](https://github.com/projectatomic/papr/blob/662cf4817893339662528c75c9cf46872bdea7bd/docs/sample.papr.yml#L106) and hook up Homu to this repo? Even though it's only a linter, hooking up Homu still gets you e.g. auto-squashing, delegation, etc..

**Comment by miabbott** [comment]
@mike-nguyen any reservations about using Homu?  Seems like we are overdue.

**Comment by mike-nguyen** [comment]
Not at all.  We are like barbarians pushing this squash and merge button.

**Comment by miabbott** [comment]
I just ran `yamllint` with the defaults against the `tests` directory and there is a mountain of warnings/errors to address.  So before we turn on the linter, let's make sure we have the existing violations sorted out.

**Comment by mike-nguyen** [comment]
#302 Addressed many of the yamllint errors but there were some lines that couldn't be easily shortened to the 80 character limit.  We could setup our own yamllint rules for make exceptions for line length.

**Comment by miabbott** [comment]
Probably need something like this in a `yamllint.cfg` file:

```yaml
---
extends: default

rules:
  linelength: 90
```

**Comment by mike-nguyen** [comment]
I found this to work.  Noting for future reference.

---                                                                                                              
extends: default

rules:
  line-length:
    max: 90
